### ISSUE TITLE
Easy feedback on issues

### ISSUE THREAD

**Issue by normanbrobinson** [18F/pulse#520]
Please provide a point of contact email for in the error page personnel to provide feedback on the errors and issues found. E.g., https://staging.pulse.cio.gov/accessibility/domain/state.gov#OtherErrors should have an email address to provide GSA feedback to pulse@gsa.gov on the issue(s) found. 

Rationale: Many coordinators and agencies may have change in personnel or people with no knowledge of the operations of the programs, and will find the correct point of contact when they view the report or pulse output.

**Comment by gbinal** [comment]
I see your point about how some folks will just circulate an individual site's error page ([here's a different one](https://staging.pulse.cio.gov/accessibility/domain/acquisition.gov#HTMLAttributeErrors) that has errors in it) and that we should think about what the user experience is for that person. 

There's a prominent feedback link in the navbar and the contact info can be found in the `About` page, but if we want to do this, I'd suggest that it be something we add to the top left of the site, where the update date is current (screenshot below).  

@smarina04 - any thoughts on this?  

![screen shot 2016-09-13 at 2 11 12 pm](https://cloud.githubusercontent.com/assets/633088/18485758/255b271c-79bc-11e6-9617-5e981ed54842.png)

**Comment by normanbrobinson** [comment]
I don't think it is simply about them circulating or sharing the results, but how they might not know where to start. Even the feedback link can be difficult on those unfamiliar and uncomfortable with github. We have to support all types of individuals who might want to submit a new issue. 

In a more perfect user experience, we might walk them through submitting a new issue. Perhaps we can help and streamline a bit by instructing them: "Problem with these results? Submit a new issue (https://github.com/18F/pulse/issues/new)"

Don't forget we need to update https://staging.pulse.cio.gov/about/#send-feedback with that related information.
